package.json

```diff
@@ -54,17 +54,20 @@
     "@reduxjs/toolkit": "^1.4.0",
     "@shopify/restyle": "^1.3.1",
     "@types/lodash": "^4.14.165",
+    "date-fns": "^2.16.1",
     "expo-haptics": "^8.3.0",
     "expo-secure-store": "^9.2.0",
     "i18next": "^19.8.3",
     "lodash": "^4.17.20",
     "lottie-ios": "3.1.8",
     "lottie-react-native": "^3.5.0",
     "react": "16.13.1",
+    "react-async-hook": "^3.6.2",
```

Comment:
this is cool! hate doing async functions in `useEffect`

Reply:
same. It gets annoying and feels misplaced.